docs/api.rst

```diff
@@ -343,3 +343,25 @@ Record Class
         database. Its value can be either a a dictionary structure which will be
         converted to JSON (e.g. for ``info(Q:group, {...})``) or something else
         which will be double quoted (e.g. for ``info(autosaveFields, "VAL")``).
+
+
+Using other dbCore functions
+----------------------------
+
+Advanced usage may require using other functions from the dbCore library to 
+get extra introspection information about records.
+
+..  py:currentmodule:: epicsdbbuilder.mydbstatic
+..  function:: GetDbFunction(name)
+
+    This can be used to get other ``dbCore`` functions for advanced usage.  Note
+    that before use their argtypes will need to be declared in the standard
+    `ctypes` way.
```

Comment:
Hmmm.  How is the user expected to apply these argtypes?  Could the necessary parameters be passed as optional arguments to `GetDbFunction`?